PixaView/Controller/DetailedViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  DetailedViewController.swift
+//  PixaView
+//
+//  Created by Дмитрий Игнатьев on 02.12.2021.
+//
+
+import UIKit
+import TestFramework
+
+class DetailedViewController: UIViewController{
+    
+    //MARK: Outlets-
+    @IBOutlet weak var detailedCollectionView: UICollectionView!
+    
+    
+    //MARK: Properties-
+    var detailedPictures: UIImage?
+    var tags = ""
+    var urlData: String?
+    
+    //MARK: viewDidLoad-
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.detailedCollectionView.register(UINib(nibName: DetailedCell.detailCellIdentifier, bundle: nil), forCellWithReuseIdentifier: DetailedCell.detailCellIdentifier)
```

Comment:
Too lengthy line you could replace it with

`self.detailedCollectionView.register(
       UINib(nibName: DetailedCell.detailCellIdentifier, bundle: nil),
       forCellWithReuseIdentifier: DetailedCell.detailCellIdentifier
)`